tests/authentication/test_authentication.py

```diff
@@ -0,0 +1,51 @@
+from allure import description, step, title, feature
+from hamcrest import assert_that, is_, is_not, empty, has_key, has_length
+
+from framework.endpoints.authenticate_api import AuthenticateAPI
+from framework.endpoints.users_api import UsersAPI
+from framework.tools.generators import generate_user_data
+
+
+@feature("Authentication of user")
+class TestAuthentication:
+    @title("Test authentication")
+    @description(
+        "GIVEN user is registered"
+        "WHEN user submit valid credential "
+        "THEN status HTTP CODE = 200 and get JWT token"
```

Comment:
`200` or `201`?

Reply:
for authentication status code = 200
for registration = 201
In my test I verify for successful authentication of a user that already exists and the expected code should be 200.